wgsl/index.bs

```diff
@@ -6916,7 +6916,7 @@ That's not a full user-defined function declaration.
     (OpAny)
 
   <tr algorithm="scalar select">
-    <td>|T| is a scalar
+    <td>|T| is a scalar or a vector
```

Comment:
As @jrprice  noted ([in a review comment](https://github.com/gpuweb/gpuweb/pull/2084#discussion_r703945528)), this fixes an accidental removal.